monai/inferers/inferer.py

```diff
@@ -157,6 +160,8 @@ def __init__(
         self.postprocessing = postprocessing
 
         # batch size for patches
+        if batch_size < 1:
```

Comment:
I don't find in which case there will be a `batch_size` less than 1; instead, it seems more likely that a batch size would be missing. What do you think?

Reply:
It's just a simple sanity check. `batch_size` is an input argument so it can't be missing but the user can mistakenly provide a wrong number less than one.